drivers/dac/ad9144/ad9144.h

```diff
@@ -797,6 +797,7 @@
 #define SEL_SIDEBAND				(1 << 1) /* 1 = Select upper or lower sideband from modulation result */
 #define I_TO_Q					(1 << 0) /* 1 = send I datapath into Q DAC */
 #define MODULATION_TYPE(x)			(((x) & 0x3) << 2) /* selects type of modulation operation */
+#define MODULATION_TYPE_MASK			(0x03 << 2)
```

Comment:
nitpick: this is not needed, you can use `MODULATION_TYPE(~0)` or something along the lines.

Reply:
If i will use the MODULATION_TYPE macro it will look like

reg = (reg & ~MODULATION_TYPE(4)) | modulation_type;

It may look a bit confusing